sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -2791,56 +2791,6 @@ public void testTimestampLiteralWithUTCTimeZone() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
-  @Test
```

Comment:
Moving these tests to proper location without changing the content.

Reply:
What is the proper location?

Reply:
Oh, just moving it to a block of related tests.